javascript/react_js/project_shopping_cart.md

```diff
@@ -15,5 +15,5 @@ This application should also have at least two routes to give you a chance to ma
 7. Once a user has submitted their order, the amount on the cart itself should adjust.
 8. Make sure to test your app thoroughly using the React Testing Library. Take care that you don't directly test `react-router-dom`, since it is an external library and the developers working on it must have tested the library already.
 9. As usual, style your application so you can show it off!
-10. Lastly, push the project to GitHub! Follow [this link](https://create-react-app.dev/docs/deployment/#github-pages) for instructions on how to deploy React applications with client-side routing to Github Pages. Be aware that the page will stay blank, unless you import { HashRouter } from react-router-dom and utilize a '<HashRouter />' component in place of '<BrowserRouter />' as GitHub no longer supports '<BrowserRouter />'. For a more detailed description, read these parts of the React Router Docs on [client-side routing](https://create-react-app.dev/docs/deployment/#notes-on-client-side-routing) and [deploying using gh-pages](https://create-react-app.dev/docs/deployment/#github-pages).
+10. Lastly, push the project to GitHub! Follow [this link](https://create-react-app.dev/docs/deployment/#github-pages) for instructions on how to deploy React applications with client-side routing to Github Pages. Be aware that the page will stay blank, unless you import { HashRouter } from react-router-dom and utilize a `<HashRouter />` component in place of `<BrowserRouter />` as GitHub no longer supports `<BrowserRouter />`. For a more detailed description, read these parts of the React Router Docs on [client-side routing](https://create-react-app.dev/docs/deployment/#notes-on-client-side-routing) and [deploying using gh-pages](https://create-react-app.dev/docs/deployment/#github-pages).
```

Comment:
Should we codify `{ HashRouter }` or even `import { HashRouter }` while we're here? 

Otherwise LGTM, good catch :ok_hand: